kvmagent/kvmagent/key_agent/key_agent_pb2_grpc.py

```
@@ -0,0 +1,152 @@
+# -*- coding: utf-8 -*-
+# 与 Go key-manager key-agent 一致：KeyAgentService, CreateEnvelopeKey, RotateEnvelopeKey, GetPublicKey, CheckEnvelopeKey, EnsureSecret
```

Ruff (0.15.2)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)